includes/Admin/Admin_AJAX.php

```diff
@@ -59,55 +60,22 @@ public function get_checks_to_run() {
 			wp_send_json_error( $valid_nonce, 403 );
 		}
 
-		$checks = filter_input( INPUT_POST, 'checks', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+		$checks = wp_parse_list( filter_input( INPUT_POST, 'checks', FILTER_SANITIZE_STRING ) );
```

Comment:
That's a fair point. But yeah, since it's definitely possible to pass query or body parameters as an array (via `param[]=...` notation), I would imagine this can be done via `FormData` as well.